control/control.go

```diff
@@ -368,6 +369,11 @@ func (c *Controller) Solve(ctx context.Context, req *controlapi.SolveRequest) (*
 		} else {
 			exp.CacheExportMode = exportMode
 		}
+		if ignoreError, supported := parseCacheExportIgnoreError(e.Attrs["ignore-error"]); !supported {
+			bklog.G(ctx).Debugf("skipping invalid cache export ignore-error: %s", e.Attrs["ignore-error"])
```

Comment:
iiuc then empty value produces a debug log. That should not be the case, only real invalid values should do it.

Reply:
yes true, I've added something more to assure that it's only printed when a not empty and not supported value is provided